index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+
+<html lang="zxx" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./styles/stylesheet.css">
+    <title>Adopt A Dog</title>
+  </head>
+  <body>
+    <header>
+
+      <figure class="company-logo">
+        <img src="images/adoptadog-logo.svg" alt="Company Logo">
+      </figure>
+
+      <figure class="cart-icon">
+        <img src="images/cart-icon.svg" alt="Cart Icon">
+      </figure>
```

Comment:
Make sure to add the checkout button so there is a way to navigate to your checkout page